### PR TITLE
Update akenza connector

### PR DESCRIPTION
**Pull request by SavinNiederer**
#### Summary

Updates the akenza webhook to work with the new akenza platform.

#### Changes

- Updates the akenza webhook to work with the new akenza platform.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [x] Testing: The changes are tested.
- [ ] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

File: akenza.yml [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+  - id: uplink_secret
+    name: Uplink Secret
+    description: Akenza uplink secret
+    secret: true
+  - id: akenza_url
+    name: Base URL
+    description: Akenza base URL
     secret: false
+    default-value: https://data-gateway.akenza.io
+format: json
```
Comment (mariusgiger) [inline akenza.yml:17]: format json is added twice

Comment (nejraselimovic) [reply akenza.yml:17]: `create-downlink-api-key` as well. Did you test this template?

**Review by nejraselimovic — CHANGES_REQUESTED** [review]
I just tested this template using a local TTS instance and Postman simultaneously, and I found an issue.

If you use `https://data-gateway.akenza.io/v3/capture?secret=<secret-value>` the data doesn't show up on akenza. On the other hand, if you use `https://data-gateway.akenza.io/v3/capture?secret=<secret-value>&deviceId=<device-id>` or `https://data-gateway.akenza.io/v3/capture?secret=<secret-value>&deviceId=<device-id>&topic=<topic-name>`, in both cases data shows up on akenza.

The `base-url` in this template is defined as `https://data-gateway.akenza.io/v3/capture?secret={uplink_secret}`, i.e. it's missing the `deviceId` part to make it work. However, if you add that part and define `https://data-gateway.akenza.io/v3/capture?secret={uplink_secret}&deviceId={device_id}`, the predefined maximum base URL length is exceeded.

I guess it will be necessary to make the endpoint URL format shorter? Your thoughts @adriansmares?

**Comment by adriansmares** [comment]
The 64 character limitation applies to message paths (`uplink_message`), but not to `base_url`. So it is entirely possible to put the full URL there for now, and to use just `""` as a message path.

**Comment by SavinNiederer** [comment]
> If you use `https://data-gateway.akenza.io/v3/capture?secret=<secret-value>` the data doesn't show up on akenza. On the other hand, if you use `https://data-gateway.akenza.io/v3/capture?secret=<secret-value>&deviceId=<device-id>` or `https://data-gateway.akenza.io/v3/capture?secret=<secret-value>&deviceId=<device-id>&topic=<topic-name>`, in both cases data shows up on akenza.
> 
> The `base-url` in this template is defined as `https://data-gateway.akenza.io/v3/capture?secret={uplink_secret}`, i.e. it's missing the `deviceId` part to make it work. However, if you add that part and define `https://data-gateway.akenza.io/v3/capture?secret={uplink_secret}&deviceId={device_id}`, the predefined maximum base URL length is exceeded.

The `deviceId` and `topic` are taken from the uplink body. I verified that the URL is correct using our current implementation which uses the default webhook instead of the akenza template.
image

Sent test uplink
image

uplink received on akenza
image

Here some documentation links to recreate this:
https://docs.akenza.io/get-started/your-integration/the-things-network
https://docs.akenza.io/get-started/your-data-flow
https://docs.akenza.io/tutorials/add-devices/how-to-integrate-a-decentlab-sensor-with-ttn-on-akenza select passthrought as a device type and send the uplink manually from TTN side

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**